Policies/Kubernetes/allowed-host-paths/azurepolicy.json

```diff
@@ -27,20 +27,43 @@
         "type": "Array",
         "metadata": {
           "displayName": "Namespace exclusions",
-          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation."
         },
         "defaultValue": ["kube-system", "gatekeeper-system", "azure-arc"]
       },
       "allowedHostPaths": {
         "type": "Object",
         "metadata": {
-          "displayName": "Valid hostPaths",
-          "description": "Configure what hostPaths are valid for volumes to use. An empty list means there is no restriction on host paths used. This is defined as a list of objects with a single pathPrefix field, which allows hostPath volumes to mount a path that begins with an allowed prefix, and a readOnly field indicating it must be mounted read-only. E.g. { \"paths\": [{\"pathPrefix\": \"/foo\", \"readOnly\": true}] }"
+          "displayName": "Allowed host paths",
+          "description": "Configure what hostPaths are valid for volumes to use. An empty list means there is no restriction on host paths used.",
+          "schema": {
+            "type": "object",
+            "properties": {
+              "paths": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "pathPrefix": {
+                      "type": "string"
+                    },
+                    "readOnly": {
+                      "type": "boolean"
+                    }
+                  },
+                  "required": ["pathPrefix", "readOnly"],
+                  "additionalProperties": false
+                }
+              }
+            },
+            "required": ["paths"],
```

Comment:
"paths" is not required as an empty list is used to block all (when we update the rego)